phpunit-integration-legacy.xml

```diff
@@ -6,6 +6,7 @@
          convertErrorsToExceptions="true"
          convertNoticesToExceptions="true"
          convertWarningsToExceptions="true"
+         beStrictAboutTestsThatDoNotTestAnything="false"
```

Comment:
Rather than using this flag, why not refactor `Netgen\EzPlatformSiteApi\Tests\Integration\RelationServiceTest` to replace the offending test with a private method that is not a test itself?